apps/sms/views/shared/js/activity_handler.js

```diff
@@ -375,12 +373,10 @@ var ActivityHandler = {
             contacts[0].name.length && contacts[0].name[0] ||
             message.sender;
 
-          var titlePromise = message.type === 'sms' ?
-            Promise.resolve(message.body || '') : getTitleFromMms();
+          var titleL10n = message.type === 'sms' ?
+            {raw: message.body || ''} : getTitleFromMms();
```

Comment:
nit: please put spaces before/after the curly brackets

also, you need a promise; you could do something like:
```javascript
Promise.resolve(titleL10n).then((titleL10n) => continueWithNotification(sender, titleL10n));
```

If you do that, you don't need to change `getTitlefromMms` to always return a Promise